maven/src/org/netbeans/modules/maven/nodes/DependencyNode.java

```diff
@@ -785,6 +785,7 @@ private Project getDependencyProject() {
                 return null;
             }
             URI uri = org.openide.util.Utilities.toURI(art.getFile());
+            // TODO should be patched, along with other uses of Artifact.getFile among nodes
```

Comment:
What are your plans with these TODOs, @jglick?

Reply:
No immediate plans. I am actually working on [eliminating uses of timestamped snapshots in Jenkins](https://github.com/jenkinsci/jep/blob/master/jep/305/README.adoc). Here I am just recording something else that seemed wrong, but was a much lower priority as it only affects display.

Reply:
OK.